src/pykeen/nn/emb.py

```diff
@@ -192,6 +192,35 @@ class Embedding(RepresentationModule):
 
     This class provides the same interface as :class:`torch.nn.Embedding` and
     can be used throughout PyKEEN as a more fully featured drop-in replacement.
+
+    It extends it by adding additional options for normalizing, constraining, or applying dropout.
+
+    When a *normalizer* is selected, it is applied in every forward pass. It can be used, e.g., to ensure that the
+    embedding vectors are of unit length. A *constrainer* can be used similarly, but it is applied after each parameter
+    update (using the post_parameter_update hook), i.e., outside of the automatic gradient computation.
+
+    The optional dropout can be used as a regularization technique. Moreover, it enables to obtain uncertainty estimates
+    via techniques such as Monte-Carlo dropout. The following simple example shows how to obtain different scores
```

Comment:
reference to Monte-Carlo dropout please

Reply:
This reference might be useful: https://arxiv.org/abs/1506.02142

Reply:
added to docstring in 6a44448